exts/cesium.usd.plugins/schemas/cesium_schemas.usda

```diff
@@ -436,29 +460,39 @@ class CesiumImageryPrim "CesiumImageryPrim" (
         doc = "The access token to use to access the Cesium ion resource. Overrides the default token. Blank if using URL."
     )
 
-    bool cesium:showCreditsOnScreen = false (
+    rel cesium:ionServerBinding (
         customData = {
-            string apiName = "showCreditsOnScreen"
+            string apiName = "ionServerBinding"
         }
-        displayName = "Show Credits On Screen"
-        doc = "Whether or not to show this imagery layer's credits on screen."
+        displayName = "Cesium Ion Server Binding"
+        doc = "Specifies which Cesium ion Server prim to use for this tileset."
     )
+}
 
-    float cesium:alpha = 1.0 (
+class CesiumPolygonImageryPrim "CesiumPolygonImageryPrim" (
+    doc = """Adds a prim for representing a polygon imagery layer. Should be a child of a tileset."""
+    inherits = </CesiumImageryPrim>
+    customData = {
+        string className = "PolygonImagery"
+    }
+) {
+    rel cesium:cartographicPolygonBinding (
         customData = {
-            string apiName = "alpha"
+            string apiName = "cartographicPolygonBinding"
         }
-        displayName = "Alpha"
-        doc = "The alpha blending value, from 0.0 to 1.0, where 1.0 is fully opaque."
+        displayName = "Cartographic Polygon Binding"
+        doc = "Specifies which Cartraphic Polygons to use in the imagery layer"
     )
+}
 
-    rel cesium:ionServerBinding (
-        customData = {
-            string apiName = "ionServerBinding"
-        }
-        displayName = "Cesium ion Server Binding"
-        doc = "Specifies which Cesium ion Server prim to use for this tileset."
-    )
+class CesiumCartographicPolygonPrim "CesiumCartographicPolygonPrim" (
+    doc = """Adds a prim that represents a Cesium CartographicPolygon"""
```

Comment:
I went with "Adds a prim for representing a globe-anchored polygon" because Basis Curves isn't great for representing these polygons. In a perfect world, we either constrain its modes with UI or drop in our own way to defining polygon verts.